scripts/pages/index.page.js

```diff
@@ -37,10 +39,32 @@ const _onSelect = function(value) {
   _setMap();
 };
 
+const _getPosts = () => {
+  const postList = document.querySelector('.js-posts-grid > .wrap');
+
+  d3_json('homepage/posts.json', (err, data) => {
```

Comment:
Why didn't you use fetch/whatwg-fetch + JSON.parse ? This will add an unnecessary dependency to D3 in the index bundle.